RocketChat/Rocket.Chat.ReactNative
# [FIX] User status update

## Proposed changes

Based on #3004
Currently we're updating the user status two times: when we tap on the user status and when we tap on `Done`. This fixes the first one, since it updates the user status even if you tap on `Cancel`
## Issue(s)	

Closes #3003
## How to test or reproduce


Test case 1

1. Go to Sidebar > Edit user status
2. Change user status / user status text
3. Tap on `Cancel`

Expected behavior: User status / user status text should be the same.

Test case 2

1. Go to Sidebar > Edit user status
2. Change user status / user status text
3. Tap on `Done`

Expected behavior: User status / user status text should be updated.

## Screenshots

## Types of changes




## Checklist



## Further comments
